app/common/SessionKeys.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package common
+
+object SessionKeys {
+  val clientVRN: String = "CLIENT_VRN"
```

Comment:
Sorry what do you mean by this?

Reply:
Didn't realise it would only show the one line. The other string is  "redirectUrl". One's all upper and this one is camel...

Unless it's supposed to be like that for some reason?

Reply:
Okay, I get what you meant now. I'll make the change